src/main/java/htsjdk/variant/vcf/VCFHeader.java

```diff
@@ -223,10 +226,8 @@ public void setSequenceDictionary(final SAMSequenceDictionary dictionary) {
         }
         mMetaData.removeAll(toRemove);
```

Comment:
@cmnbroad This method also iterates over all the headerlines, as do some others, do you think we should clean them up now or wait for your big change?